Common/cpp/Tools/RuntimeDecorator.cpp

```diff
@@ -10,6 +10,11 @@
 
 namespace reanimated {
 
+static const std::function<jsi::String(jsi::Runtime &, jsi::Value const &)>
+    stringifyValue = [](jsi::Runtime &rt, jsi::Value const &value) {
+      return jsi::String::createFromUtf8(rt, stringifyJSIValue(rt, value));
+    };
```

Comment:
could you check if something like this will work? I guess it should and it's cleaner. Same for next function

```suggestion
static jsi::String stringifyValue(jsi::Runtime &rt, jsi::Value const &value) {
      return jsi::String::createFromUtf8(rt, stringifyJSIValue(rt, value));
};
```